third-party/cmake/BuildLuv.cmake

```diff
@@ -86,7 +86,11 @@ else()
     "-DCMAKE_C_FLAGS:STRING=${CMAKE_C_COMPILER_ARG1} ${LUV_INCLUDE_FLAGS} -fPIC")
 endif()
 
-set(LUV_BUILD_COMMAND ${CMAKE_COMMAND} --build .)
+set(EXTRA_BUILD_FLAGS "")
+if(CMAKE_SYSTEM_NAME MATCHES "OpenBSD")
+  string(APPEND EXTRA_BUILD_FLAGS "-DCMAKE_MAKE_PROGRAM=gmake")
```

Comment:
Might as well do this for all BSDs. Also just do `set(LUV_BUILD_COMMAND ...)` in if-else, instead of a new `EXTRA_BUILD_FLAGS` variable.

Reply:
done!